search-provider/eks-discovery-feed-provider.c

```diff
@@ -278,9 +377,15 @@ handle_article_card_descriptions (EksDiscoveryFeedDatabaseContentProvider *skele
     g_variant_builder_add (&tags_match_any_builder, "s", "EknArticleObject");
     GVariant *tags_match_any = g_variant_builder_end (&tags_match_any_builder);
 
+    GVariantBuilder tags_match_all_builder;
+    g_variant_builder_init (&tags_match_all_builder, G_VARIANT_TYPE ("as"));
+    g_variant_builder_add (&tags_match_all_builder, "s", "EknHasDiscoveryFeedTitle");
+    GVariant *tags_match_all = g_variant_builder_end (&tags_match_all_builder);
```

Comment:
It seems like the constructor for `EkncQueryObject` consumes the value. I put it behind g_autoptr and got warnings about the ref count going below 1.